Detectors/HMPID/reconstruction/src/HmpidDecoder.cxx

```diff
@@ -89,8 +89,8 @@ HmpidDecoder::~HmpidDecoder()
 /// Init all the members variables.
 void HmpidDecoder::init()
 {
-  mRDHAcceptedVersion = 6;
-  mRDHSize = sizeof(o2::header::RAWDataHeaderV6) / sizeof(uint32_t);
+  mRDHAcceptedVersion = 7;
+  mRDHSize = sizeof(o2::header::RAWDataHeaderV7) / sizeof(uint32_t);
```

Comment:
I don't see where mRDHAcceptedVersion is used, but we should be able to work also with V6. Better use generic `o2::header::RAWDataHeader` which is an alias to default RDH in use. In case you need the nominal default version, you can use
```
o2::header::RAWDataHeader rdh;
mRDHAcceptedVersion = rdh.version;
mRDHSize = rdh.headerSize;
```